src/runtime_src/core/pcie/driver/linux/xocl/subdev/xmc.c

```diff
@@ -2710,6 +2723,8 @@ static int load_xmc(struct xocl_xmc *xmc)
 				xmc->mgmt_binary_length);
 			COPY_MGMT(xmc, xmc->mgmt_binary, xmc->mgmt_binary_length);
 		}
+	} else {
+		xocl_info(&xmc->pdev->dev, "Skip copying XMC image");
```

Comment:
I will add more info "Skipping loading xmc image since xmc is already part of bitfile"